src/main/kotlin/org/rucca/cheese/common/pagination/util/ReflectionUtils.kt

```
@@ -0,0 +1,63 @@
+                if (property == null) {
+                    // Property not found on the current object's class
+                    // Log warning or handle as needed
+                    // logger.warn("Property '$part' not found on class ${currentClass.simpleName}
+                    // for path '$path'")
+                    return null
+                }
+
+                // Get the value of the property for the current object
+                current = property.get(current!!)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)